dev/com.ibm.ws.security.wim.core/src/com/ibm/ws/security/wim/ConfigManager.java

```diff
@@ -496,4 +510,56 @@ public long getPageCacheTimeOut() {
     void setRepositoryManager(RepositoryManager repositoryManager) {
         this.repositoryManager = repositoryManager;
     }
+
+    @Trivial
+    @Reference(service = FutureMonitor.class)
+    protected void setFutureMonitor(FutureMonitor futureMonitor) {
+        _futureMonitor = futureMonitor;
+    }
+
+    @Trivial
+    protected void unsetFutureMonitor(FutureMonitor futureMonitor) {
+        _futureMonitor = null;
+    }
+
+    /**
+     * After a configuration modification, run a RealConfigChange check which will verify the participatingBaseEntries.
+     *
+     * We used to do this directly from the modify command, but timing change during startup/modify and we would process the check
+     * before the user registries activated, leading to misleading error messages being printed.
+     */
+    @Trivial
+    @Override
+    public void notificationCreated(RuntimeUpdateManager updateManager, RuntimeUpdateNotification notification) {
+        if (modified && notification.getName().equals(RuntimeUpdateNotification.CONFIG_UPDATES_DELIVERED)) {
```

Comment:
Reverse the string equality check so the constant is first to protect against possible NPE.

Reply:
Done